docs/developer/available-plugins/user-emails.mdx

```diff
@@ -309,7 +311,7 @@ Event name: _order_confirmation_
 ```json
 {
   "order": {
-    "id": 51,
+    "id": "VXNlcjo1MQ==",
```

Comment:
that has `User` type encoded while it's an `Order` (relates to entire pr). If as a dev I read these docs and wonder what actually hides behind this magic ID, I'd immediately spot this looks like b64, decode it and get confused ;D